automod/rules/reposts.go

```diff
@@ -0,0 +1,42 @@
+package rules
+
+import (
+	"time"
+
+	"github.com/bluesky-social/indigo/automod"
+	"github.com/bluesky-social/indigo/automod/countstore"
+)
+
+var repostDailyThreshold = 500
```

Comment:
Definitely curious if we'll have any triggers at this value! I might have started at 100... but maybe these accounts are going really hard already? And there might be heavy organic re-post use by some users.

Reply:
good point, at least for test, I can set a lower threshold maybe?